### PR TITLE
release 4.8.1

### PR DESCRIPTION
**Pull request by northwestwitch**
New Scout release 4.8.1

### Fixed
- Prevent login fail for users with id defined by ObjectId and not email
- Prevent the app from crashing with `AttributeError: 'NoneType' object has no attribute 'message'`

**Review:**
- [x] tests executed by DN, CR
- [x] "merge and deploy" approved by @ingkebil 

This is a minor release because provides two bug fixes.

### CODE REVIEW

File: scout/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '4.8.0'
+__version__ = '4.8.1'
```
Comment (ingkebil) [inline scout/__version__.py]: So it's a patch and not a minor? :)

Comment (northwestwitch) [reply scout/__version__.py]: Oh yes, sorry, I meant a patch!

**Comment by northwestwitch** [comment]
@ingkebil could you approve?

**Comment by northwestwitch** [comment]
Thanks!! :)

**Comment by northwestwitch** [comment]
I'm planning to deploy it tonight, only on clinical DB @emiliaol @dnil @moonso

**Pull request merged**